test/types/paypal.ts

```diff
@@ -0,0 +1,23 @@
+export default function paypal() {
+  const paypal = window.recurly.PayPal();
```

Comment:
Might add some testing for the `options` that can be passed in to the constructor here.